src/rust/ide/view/src/documentation.rs

```diff
@@ -162,25 +159,33 @@ impl ViewModel {
     fn append_listeners_to_copy_buttons(&self) {
         let code_blocks  = self.dom.dom().get_elements_by_class_name("CodeBlock");
         let copy_buttons = self.dom.dom().get_elements_by_class_name("copyCodeBtn");
-        for i in 0..copy_buttons.length() {
-            let copy_button = copy_buttons.get_with_index(i);
-            let code_block  = code_blocks.get_with_index(i);
-            copy_button.map(|btn| { btn.dyn_into::<HtmlElement>().map(|cpy_btn| {
-                code_block.map(|code| { code.dyn_into::<HtmlElement>().map(|code_blk| {
-                    let closure  = move |_event: MouseEvent| {
-                        let inner_code = code_blk.inner_text();
-                        clipboard::write_text(inner_code);
-                    };
-                    let closure  = Closure::wrap(Box::new(closure).into_fn_mut());
-                    let callback = closure.as_ref().unchecked_ref();
-                    match cpy_btn.add_event_listener_with_callback("click", callback){
-                        Ok(_)    => (),
-                        Err(err) => error!(&self.logger, "Unable to add event listener: {err:?}")
+        (0..copy_buttons.length()).map(|i| -> Option<Closure<dyn std::ops::FnMut(web_sys::MouseEvent)>>{
+            let copy_button = copy_buttons.get_with_index(i)?;//.dyn_into::<HtmlElement>()?;
+            let code_block  = code_blocks.get_with_index(i)?;//.dyn_into::<HtmlElement>()?;
```

Comment:
```suggestion
            let copy_button = copy_buttons.get_with_index(i)?.dyn_into::<HtmlElement>().ok()?;
            let code_block  = code_blocks.get_with_index(i)?.dyn_into::<HtmlElement>().ok()?;
```

Reply:
I've already made it, you must've reviewed some old commit haha